pallets/runtime/tests/src/external_agents_test.rs

```diff
@@ -247,3 +250,69 @@ fn add_works() {
         assert_noop!(accept(dave, id), Error::NumFullAgentsOverflow);
     });
 }
+
+fn create_asset(ticker: &[u8], owner: User) -> Ticker {
+    let (ticker, token) = token(ticker, owner.did);
+    assert_ok!(basic_asset(owner, ticker, &token));
+    ticker
+}
```

Comment:
So `create_asset` already exists in `asset_test`; let's import that one instead.